UI/src/view/linear_asset/trSpeedLimitStyle.js

```diff
@@ -89,7 +89,7 @@
     me.browsingStyleProvider.addRules(oneWayOverlayStyleRules);
 
     this.isUnknown = function(linearAsset) {
-      return !_.isNumber(linearAsset.value);
+      return !_.isNumber(linearAsset.value.value);
```

Comment:
Can this "value.value" be changed to something more explicit?

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code